pkg/repo/repo.go

```diff
@@ -134,9 +135,24 @@ func (r *RepoFile) Remove(name string) bool {
 
 // WriteFile writes a repositories file to the given path.
 func (r *RepoFile) WriteFile(path string, perm os.FileMode) error {
+	f, err := atomicfile.New(path, os.FileMode(0666))
```

Comment:
Hum, you're right... this blows up when I use it though, I need to investigate what's going on:
```
go test pkg/repo/repo_test.go pkg/repo/repo.go pkg/repo/chartrepo.go pkg/repo/index.go
--- FAIL: TestWriteFile (0.00s)
	repo_test.go:210: failed to load file (open /var/folders/57/dtb_m3856_15jd_1t_kpwfgr0000gn/T/helm-repo943240869: permission denied)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x23f6b96]

goroutine 34 [running]:
testing.tRunner.func1(0xc4206c45b0)
	/usr/local/Cellar/go/1.8.1/libexec/src/testing/testing.go:622 +0x29d
panic(0x2558ec0, 0x3278370)
	/usr/local/Cellar/go/1.8.1/libexec/src/runtime/panic.go:489 +0x2cf
command-line-arguments.(*RepoFile).Has(0x0, 0x271deea, 0x6, 0xc4206c1f58)
	/Users/rabadin/go/src/k8s.io/helm/pkg/repo/repo.go:113 +0x26
command-line-arguments.TestWriteFile(0xc4206c45b0)
	/Users/rabadin/go/src/k8s.io/helm/pkg/repo/repo_test.go:213 +0x444
testing.tRunner(0xc4206c45b0, 0x2790590)
	/usr/local/Cellar/go/1.8.1/libexec/src/testing/testing.go:657 +0x96
created by testing.(*T).Run
	/usr/local/Cellar/go/1.8.1/libexec/src/testing/testing.go:697 +0x2ca
```